Code/Tools/AssetProcessor/native/unittests/AssetProcessorUnitTests.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+* Copyright (c) Contributors to the Open 3D Engine Project.
+* For complete copyright and license terms please see the LICENSE at the root of this distribution.
+*
+* SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+*/
+
+#include <AzCore/UserSettings/UserSettingsComponent.h>
+#include <AzCore/Settings/SettingsRegistryMergeUtils.h>
+#include <native/unittests/AssetProcessorUnitTests.h>
+#include <native/unittests/UnitTestRunner.h> // for the assert absorber.
+#include <native/tests/MockAssetDatabaseRequestsHandler.h>
+
+namespace UnitTest
+{
+    AssetProcessorUnitTestAppManager::AssetProcessorUnitTestAppManager(int* argc, char*** argv)
+        : BatchApplicationManager(argc, argv)
+    {
+    }
+
+    bool AssetProcessorUnitTestAppManager::PrepareForTests()
+    {
+        if (!ApplicationManager::Activate())
+        {
+            return false;
+        }
+
+        // Not needed for this test, so disconnect
+        AssetProcessor::AssetBuilderInfoBus::Handler::BusDisconnect();
+
+        // Disable saving global user settings to prevent failure due to detecting file updates
+        AZ::UserSettingsComponentRequestBus::Broadcast(&AZ::UserSettingsComponentRequests::DisableSaveOnFinalize);
+
+        m_platformConfig.reset(new AssetProcessor::PlatformConfiguration);

```

Comment:
possible to use aznew? such as "aznew AssetProcessor::PlatformConfiguration"

Reply:
Will make the change.